kafka/vendor/six.py

```diff
@@ -643,13 +661,16 @@ def u(s):
     import io
     StringIO = io.StringIO
     BytesIO = io.BytesIO
+    del io
```

Comment:
A new `del` statement. Does the same condition here apply as in https://github.com/dpkp/kafka-python/pull/979#discussion_r100403389?